README.md

```diff
@@ -302,21 +287,6 @@ The files under `handwritten_support` provide library definitions for
 Coq, Isabelle and HOL4.
 
 
-Directory Structure
```

Comment:
Could keep this, I don't think it adds much maintenance burden

Reply:
Yeah it's the sort of thing that people forget to update though. E.g. I didn't update it when adding the first party tests.

I also think it's of little value. The `doc` directory contains documentation. The `tests` directory contains tests, etc.

But if people really want to keep it I don't mind.

Reply:
It's a bit like forgetting to update the list of supported extensions in the README: easy to fix after the fact, and useful for newcomers.